### DISCUSSION TITLE
Is there a way to set the background color such that it remains opaque...

### DISCUSSION THREAD

**Discussion by EdmundsEcho** [catppuccin/tmux#230] [Help]
The status panel has a left and right divider.  The dividers that also host a character, e.g., middle divider, are subject to the transparency levels of the shell (tested in both iterm2 and fishshell).  In general, is there a way to tell the terminal to set a background color such that it remains opaque?  If so, I suspect this is something catppuccin would want to consider when setting the background.  Otherwise, what happens is that the background color contrasts between the dividers and the those that host a chararter.  That said, a way to get each background to match, might be to place an invisible character in the left and right dividers so that the bg therein tracks the others.

**Comment by vdbe** [comment]
Can you provide a screenshot of what you mean? As far as I have seen the text is never transparent.


As for invisible characters this is not an option since it would widen the spacers.
